tests/rptest/util.py

```diff
@@ -316,18 +316,25 @@ def inject_remote_script(node, script_name):
     return remote_path
 
 
-def get_cluster_license():
-    license = os.environ.get("REDPANDA_SAMPLE_LICENSE", None)
+def _get_cluster_license(env_var):
+    license = os.environ.get(env_var, None)
     if license is None:
         is_ci = os.environ.get("CI", "false")
         if is_ci == "true":
             raise RuntimeError(
-                "Expected REDPANDA_SAMPLE_LICENSE variable to be set in this environment"
-            )
+                f"Expected {env_var} variable to be set in this environment")
 
     return license
 
 
+def get_cluster_license():
+    return _get_cluster_license("REDPANDA_SAMPLE_LICENSE")
+
+
+def get_second_cluster_license():
+    return _get_cluster_license("REDPANDA_SECOND_SAMPLE_LICENSE")
+
```

Comment:
I guess in backporting we'll need to backport the vtools changes, or omit the new tests

Reply:
https://github.com/redpanda-data/vtools/pull/1326